empress_gui.py

```diff
@@ -0,0 +1,335 @@
+
+        # creates a logo frame on top of the master frame 
+        self.logo_frame = tk.Frame(master)
+        # sticky="nsew" means that self.logo_frame expands in all four directions (north, south, east and west) 
+        # to fully occupy the allocated space in the grid system (row 0 column 0&1)
+        self.logo_frame.grid(row=0, column=0, columnspan=2, sticky="nsew")
+        self.logo_frame.grid_propagate(False)
+
+        # adds background image
+        photo = tk.PhotoImage(file="./assets/Jane_TransLogo_thin.png")
+        label = tk.Label(self.logo_frame, image=photo)
+        label.place(x=0, y=0)
+        label.image = photo
+
+        # creates an input frame on the left side of the master frame 
+        self.func_frame = tk.Frame(master)
+        self.func_frame.grid(row=1, column=0, sticky="nsew")
+        self.func_frame.grid_rowconfigure(0, weight=1)
+        self.func_frame.grid_rowconfigure(1, weight=1)
+        self.func_frame.grid_rowconfigure(2, weight=1)
+        self.func_frame.grid_rowconfigure(3, weight=1)
+        self.func_frame.grid_columnconfigure(0, weight=1)
+        self.func_frame.grid_propagate(False)
+
+        # creates an output frame on the right side of the master frame
+        self.output_frame = tk.Frame(master)
+        self.output_frame.grid(row=1, column=1, sticky="nsew")
+        self.output_frame.grid_rowconfigure(0, weight=1)
+        self.output_frame.grid_rowconfigure(1, weight=1)
+        self.output_frame.grid_rowconfigure(2, weight=1)
+        self.output_frame.grid_columnconfigure(0, weight=1)
+        self.output_frame.grid_propagate(False)
+
+        # "Load File" button 
+        # loads in an input .newick file
+        # and displays the number of leaves in each tree (DEMO for now) and entry boxes for DTL costs
+        load_file_btn = tk.Button(self.func_frame, text="Load File", background="green", height=2, width=9, command=self.load_file)
+        load_file_btn.grid(row=0, column=0)
+        # creates a Label to overwrite the old file path 
+        self.file_path_label = tk.Label(self.output_frame)
+
+        # "View Event Cost Regions" button 
+        # pops up a matplotlib graph for the cost regions
+        self.view_cost_btn = tk.Button(self.func_frame, text="View Event Cost Regions", command=self.plot_cost_regions, state=tk.DISABLED)
+        self.view_cost_btn.grid(row=1, column=0)
+
+        # "Compute Reconciliations" button 
+        # displays reconciliation results(numbers) and three options(checkboxes) for viewing graphical analysis
+        self.compute_recon_button = tk.Button(self.func_frame, text="Compute Reconciliations", command=self.recon_analysis, state=tk.DISABLED)
+        self.compute_recon_button.grid(row=2, column=0)
+
+    def load_file(self):
+        """Loads in an input file and displays the number of leaves in each tree when "Load File" button is clicked."""
+        # creates a frame to display the number of leaves and the file path
```

Comment:
I think it might look better to use capital letter to start a block comment.

Reply:
Done:)